src/Microsoft.PowerShell.PSReadLine/ConsoleKeyChordConverter.cs

```diff
@@ -323,8 +325,19 @@ internal static char GetCharFromConsoleKey(ConsoleKey key, ConsoleModifiers modi
 
             // get corresponding character  - may be 0, 1 or 2 in length (diacritics)
             var chars = new char[2];
-            int charCount = NativeMethods.ToUnicode(
-                virtualKey, scanCode, state, chars, chars.Length, NativeMethods.MENU_IS_INACTIVE);
+            int charCount = 1;
+            if (_toUnicodeApiAvailable)
```

Comment:
After a closer look, I don't think this is the right fix for this case - custom key bindings need a valid results, you'll just return `\0`.

I think we'll need the Unix variant of this code as a fallback, or potentially use it unconditionally, though that might break some more obscure key bindings.